db/migrate/20181017213808_create_columns_for_user_table.rb

```diff
@@ -0,0 +1,10 @@
+class CreateColumnsForUserTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :columns_for_user_tables do |t|
```

Comment:
I think this ended up being a separate table instead of these getting added to the users table.